roles/wazuh/ansible-wazuh-manager/tasks/main.yml

```diff
@@ -322,8 +322,7 @@
   notify: restart wazuh-api
   when:
     - wazuh_api_user is defined
-    - not (ansible_distribution == 'CentOS' or ansible_distribution == 'RedHat' or ansible_distribution == 'Amazon')
-    - ansible_distribution_major_version|int < 6
+    - not (ansible_facts['os_family']|lower == 'redhat' and ansible_distribution_major_version|int < 6)
```

Comment:
When reviewing it just noticed that this conditiional should not exists as creating the `/var/ossec/api/configuration/auth/user` is not OS dependant, please remove it.

Reply:
Tanks @jm404 , just fixed as you suggest in my `sgargel:patch-1` branch. I have tu submit a new PR ?